testplan/report/testing/base.py

```diff
@@ -352,6 +361,26 @@ def tags_index(self):
                 *[child.tags_index for child in self])
         return self._tags_index
 
+    @property
+    def information(self):
+        if not self._info:
+            try:
+                user = getpass.getuser()
+            except (ImportError, OSError):
+                # if the USERNAME env variable is unset on Windows, this fails
+                # with ImportError
+                user = 'unknown'
+            self._info = [
+                ('user', user),
```

Comment:
why not store as a `dict` instead of a list of tuples? Seems marginally simpler.

Reply:
> why not store as a `dict` instead of a list of tuples? Seems marginally simpler.

For the order of items. If it's a dict/OrderedDict, it would be a object in JSON. I want to keep the same order on the UI side

Reply:
why does the order matter?

Reply:
> why does the order matter?

For the user experience. For example, if there are "git commit id", "git branch", "git repo" and "command line string" in the information values. Displaying "command line string", "git commit id", "git branch", "git repo" would be better than Displaying  "git commit id", "command line string", "git branch", "git repo".